frontend/src/pages/QADashboard/RecipientsWithOhsStandardFeiGoal/index.js

```diff
@@ -52,6 +56,80 @@ export default function RecipientsWithOhsStandardFeiGoal() {
     QA_DASHBOARD_FILTER_CONFIG,
   );
 
+  useDeepCompareEffect(() => {
+    async function fetchQaDat() {
+      setIsLoading(true);
+      // Filters passed also contains region.
+      try {
+        const data = await getSelfServiceData(
+          'recipients-with-ohs-standard-fei-goal',
+          filters,
+          ['with_fei_widget', 'with_fei_page'],
+        );
+
+        // Get summary and row data.
+        const pageData = data.filter((d) => d.data_set === 'with_fei_page');
+        const widgetData = data.filter((d) => d.data_set === 'with_fei_widget');
+        // Convert data to format that widget expects.
+        let formattedRecipientPageData = pageData[0].data.map((item) => {
+          const { recipientId } = item;
+          const regionId = item['region id'];
+          const { recipientName } = item;
+          const { goalId } = item;
+          const { goalStatus } = item;
+          // const { grantNumber } = item;
+          const { createdAt } = item;
+          const { rootCause } = item;
+
+          return {
+            id: recipientId,
+            heading: recipientName,
+            name: recipientName,
+            isUrl: true,
+            hideLinkIcon: true,
+            link: `/recipient-tta-records/${recipientId}/region/${regionId}/profile`,
+            data: [
+              {
+                title: 'Goal_created_on',
+                value: moment(createdAt).format('MM/DD/YYYY'),
+              },
+              {
+                title: 'Goal_number',
+                value: `G-${goalId}`,
+              },
+              {
+                title: 'Goal_status',
+                value: goalStatus,
+              },
+              {
+                title: 'Root_cause',
+                value: rootCause && rootCause.length ? rootCause.join(', ') : '', // Convert array to string.
+              },
+            ],
+          };
+        });
+
+        // Add headers.
+        formattedRecipientPageData = {
+          headers: ['Goal created on', 'Goal number', 'Goal status', 'Root cause'],
+          RecipientsWithOhsStandardFeiGoal: [...formattedRecipientPageData],
+        };
+
+        setRecipientsWithOhsStandardFeiGoal({
+          pageData: formattedRecipientPageData,
+          widgetData: widgetData[0].data[0],
+        });
+        updateError('');
+      } catch (e) {
+        updateError('Unable to fetch QA data');
```

Comment:
same comments as before, no test for this case that I can see, and the function has a typo

Reply:
added test fixed typo